core/src/main/scala/com/microsoft/ml/spark/featurize/CleanMissingData.scala

```diff
@@ -158,6 +159,8 @@ class CleanMissingDataModel(val uid: String)
 
   def setFillValues(v: Array[Any]): this.type = set(fillValues, v)
 
+  def setFillValues(v: java.util.ArrayList[Any]): this.type = set(fillValues, v.asScala.toArray)
```

Comment:
Yes we need it. We need to convert the dotnet array into java.util.ArrayList so that scala can recognize it, and a function converting scala array to java ArrayList in scala side doesn't help, so we need exactly that type passed in the function to solve the problem.. We might discuss this in detail if you have any better ideas!